src/EnergyPlus/HVACFan.cc

```diff
@@ -276,17 +276,26 @@ namespace HVACFan {
         m_rhoAirStdInit = DataEnvironment::StdRhoAir;
         m_maxAirMassFlowRate = designAirVolFlowRate * m_rhoAirStdInit;
 
-        // calculate total fan system efficiency at design
-        m_fanTotalEff = designAirVolFlowRate * deltaPress / designElecPower;
+        // calculate total fan system efficiency at design, else set to 1 to avoid div by zero
+        if (designElecPower > 0.0) {
+            m_fanTotalEff = designAirVolFlowRate * deltaPress / designElecPower;
+        } else{
+            m_fanTotalEff = 1.0;
+        }
```

Comment:
This is where the Fan:SystemModel div by zero was happening (because the unit test setup had a blank for Design Electric Power which became zero. But it's reasonable for a user to enter zero power for special cases, so this needs to fall back to an efficiency of 1.0 if that's the case, because efficiency gets used in the denominator later on.

Reply:
Seems fully reasonable to set efficiency to 1 for zero power usage.